### DISCUSSION TITLE
How to extract separate parts to different pdfs using command-line?

### DISCUSSION THREAD

**Discussion by TuHedownia** [musescore/MuseScore#16754] [Q&A]
Basically what it says in the title.
Question about MS4.
I tried using -P argument but doesnt seem to work, I am not sure if im using it correctly though.
Also tried using a json job but it doesnt produde any output. Even if I try to output a single pdf file.

Can i have some help with this?

**Comment by ghost** [comment]
I don't believe parts can be saved separately in MuseScore 4, yet; see https://github.com/musescore/MuseScore/issues/15582

**Reply by TuHedownia** [reply]
They can be exported separately to pdfs though thourgh GUI, but Im not having any succes doing it from command-line options

**Reply by ghost** [reply]
Oh true. I should learn to read

**Reply by TuHedownia** [reply]
> Oh true. I should learn to read

Should I add and Issue? Im not sure
